src/main/resources/org/gradle/api/internal/plugins/windowsStartScript.txt

```diff
@@ -17,6 +17,8 @@ if "%DIRNAME%" == "" set DIRNAME=.\
 set APP_BASE_NAME=%~n0
 set APP_HOME=%DIRNAME%${appHomeRelativePath}
 
+set "DEFAULT_JVM_OPTS=%DEFAULT_JVM_OPTS% -Dpa.scheduler.home="%APP_HOME%""
```

Comment:
I tested with the scheduler started inside a folder which contains spaces and it works